src/parser/parser.cc

```diff
@@ -1107,11 +1111,26 @@ class Parser {
           [&]() {
             auto token = Match(TokenType::kLocal);
             auto string = token.ToString();
+
+            // The fake attributes where the virtual device is specified.
+            VirtualDevice virtual_device;
+            if (WhenMatch(TokenType::kLCurly)) {
+              Map<String, ObjectRef> fake_attrs = ParseAttrs();
+              VLOG(1) << "Fake attributes for function parameter: " << fake_attrs;
```

Comment:
meganit: I've been using VLOG(9) for these super-duper verbose ones since I often just set TVM_LOG_DEBUG="DEFAULT=1" to get an overall debug trace that's still vaguely readable.